### PR TITLE
Fix credscan bugs

### PR DESCRIPTION
**Pull request by aik-jahoda**
follow up of #38026

### CODE REVIEW

File: src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs [hunk 1]
```diff
@@ -131,6 +132,7 @@ internal ECParameters ExportParameters(bool includePrivateParameters, int keySiz
         {
             // Apple requires all private keys to be exported encrypted, but since we're trying to export
             // as parsed structures we will need to decrypt it for the user.
+            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Unit test password.")]
```
Comment (vcsjones) [inline src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs:135]: This code, and a few other places, are not part of a unit test. Perhaps the comment above can be incorporated as the justification?

Comment (aik-jahoda) [reply src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs]: Unfortunately it is not possible to use multiline justification. I added short justification with reference to the source.

File: .config/CredScanSuppressions.json [hunk 2]
```diff
@@ -53,7 +53,11 @@
                 "src/libraries/System.Net.Http/tests/UnitTests/HttpEnvironmentProxyTest.cs",
                 "src/libraries/System.Net.Mail/tests/Functional/SmtpClientTest.cs",
                 "src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs",
-                "src/libraries/System.Security.Cryptography.Xml/tests/TestHelpers.cs"
+                "src/libraries/System.Security.Cryptography.Xml/tests/TestHelpers.cs",
+                "src/libraries/System.Security.Cryptography.X509Certificates/tests/PfxTests.cs",
+                "src/libraries/System.Management/src/System/Management/ManagementScope.cs",
```
Comment (ManickaP) [inline .config/CredScanSuppressions.json:58]: This file, according to the path, doesn't look like a test code. Is there a reason we cannot do inline suppression for it instead?

File: src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs [hunk 3]
```diff
@@ -118,6 +118,7 @@ private void SetKey(SecKeyPair keyPair)
 
         internal static ECParameters ExportPublicParametersFromPrivateKey(SafeSecKeyRefHandle handle)
         {
+            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Password for temporary operation. See code comments for more details.")]
```
Comment (ManickaP) [inline src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs:121]: > `See code comments for more details.`

There're no comments though.

**Review by ManickaP — APPROVED** [review]
Small comments, otherwise LGTM. Thanks!

**Comment by ericstj** [comment]
@aik-jahoda is this PR still important?  Be sure to rerun tests if you want to merge it.

**Comment by joperezr** [comment]
ping @aik-jahoda 😄 

Do we still need this PR or can we go ahead and close?

**Comment by aik-jahoda** [comment]
Closing as we have new credscan effort and we need reevaluate the approach

**Pull request closed without merging**